protoc-gen-openapiv2/internal/genopenapi/generator.go

```diff
@@ -114,6 +116,58 @@ func (so openapiSwaggerObject) MarshalYAML() (interface{}, error) {
 	}, nil
```

Comment:
I think we can make this change (see GolangCI warning above)

Reply:
I've just made the changes listed in the GolangCI warnings, hopefully everything is okay now!